Example/Example/SeriesWireframe.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ComicsWireframe.swift
+//  Example
+//
+//  Created by Davide Mendolia on 03/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+class SeriesWireframe: ExampleWireframe {
+
+    func presentSerieDetailViewController(serieName: String) {
+        let viewController = serviceLocator.provideSerieDetailViewController()
```

Comment:
you already know it, but i put here as a reminder for next steps. 
You will need to pass, serieName as param to  provideSerieDetailViewController.